x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_executed_details.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { Ecs } from '../../../../graphql/types';
+import { escapeQueryValue } from '../../../../lib/keury';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+const Details = styled.div`
+  margin-left: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+`;
+
+const MarginLeftFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+const MarginRightFlexItem = styled(EuiFlexItem)`
+  margin-right: 3px;
+`;
+
+export const DraggableAuditdExecutedElement = pure<{
+  id: string;
+  name?: string;
+  field: string;
+  value: string | null;
+  queryValue?: string | null;
+}>(({ id, name, field, value, queryValue }) =>
+  value != null ? (
+    <DraggableWrapper
+      dataProvider={{
+        and: [],
+        enabled: true,
+        id: escapeDataProviderId(`auditd-executed-element-${id}-${field}-${value}`),
+        name: name ? name : value,
+        excluded: false,
+        kqlQuery: '',
+        queryMatch: {
+          field,
+          value: escapeQueryValue(queryValue ? queryValue : value),
+        },
+      }}
+      render={(dataProvider, _, snapshot) =>
+        snapshot.isDragging ? (
+          <DragEffects>
+            <Provider dataProvider={dataProvider} />
+          </DragEffects>
+        ) : (
+          <>{value}</>
+        )
+      }
+    />
+  ) : null
+);
```

Comment:
This is more for discussion, but are we getting to a point where it makes sense for these `DraggableWrapper`'s to be abstracted further? It's looking as if a default/basic configuration is solidifying between all our draggables (mostly in tables + timeline renderers). The signature for creating a `DraggableZeekElement` from your last PR is very similar, as are some other recent additions. Even if `DraggableWrapper` had a default `dataProvider` for a given `id`/`field`/`value` seems like it would be helpful as those look pretty similar across the board. This would clean up testing around all these components in our app as well :)

This is just high-level thinking as the Timeline becomes decoupled from our app over time. We'll probably want to provide a simple interface+component for developers to create draggables that we can bundle with the Timeline, no? Thoughts? @andrew-goldstein has this been on your mind at all?

Reply:
I would say eventually yes. I also had conversations where if we see repeating patterns we should pull them out but if we end up introducing a ton of optional arguments or other branching logic, then they get broken out again.

Reply:
@spong it seems we're all seeing a few patterns emerging, like "draggable + copy to clipboard", so it does sound like it's time to have a team chat about how we compose and re-use capabilities. 😄 

I also agree with @FrankHassanabad that we want to avoid the "everything's an option" pattern.